plugins/repository-gcs/src/test/java/io/crate/gcs/GCSSnapshotIntegrationTest.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.gcs;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.net.InetAddress;
+import java.net.InetSocketAddress;
+import java.util.ArrayList;
+import java.util.Collection;
+
+import org.elasticsearch.plugins.Plugin;
+import org.elasticsearch.test.IntegTestCase;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.sun.net.httpserver.HttpServer;
+
+@IntegTestCase.ClusterScope(scope = IntegTestCase.Scope.TEST)
+public class GCSSnapshotIntegrationTest extends IntegTestCase {
+
+    private static final String PKCS8_PRIVATE_KEY = """
```

Comment:
Maybe add a comment explaining how this was generated.

Reply:
Yes I figured that it is an arbitrary key. The command would still be nice to have in case we need to change the key in the future - e.g to raise the bit length or something.